From 4217cb5e07519ea41fdff9b906b93ce6946c109e Mon Sep 17 00:00:00 2001
From: Colin Walters <walters@verbum.org>
Date: Sat, 30 Jul 2011 15:40:25 +0000
Subject: Install gi/ headers again

libpeas currently requires these; we were installing them before, and
since we export almost all of our internals in for gjs-module.pc, might
as well keep doing these for now.

https://bugzilla.gnome.org/show_bug.cgi?id=655482
---
diff --git a/Makefile.am b/Makefile.am
index edf0c25..b088289 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -36,7 +36,20 @@ nobase_gjs_module_include_HEADERS =	\
 	gjs/importer.h		\
 	gjs/jsapi-util.h	\
 	gjs/mem.h		\
-	gjs/native.h
+	gjs/native.h	\
+	gi/ns.h	        \
+	gi/object.h	\
+	gi/foreign.h	\
+	gi/param.h	\
+	gi/repo.h	\
+	gi/union.h	\
+	gi/value.h	\
+	gi/arg.h	\
+	gi/boxed.h	\
+	gi/closure.h	\
+	gi/enumeration.h	\
+	gi/function.h	\
+	gi/keep-alive.h	
 
 noinst_HEADERS +=		\
 	gjs/jsapi-private.h	\
@@ -108,20 +121,7 @@ libgjs_la_SOURCES =		\
 
 # For historical reasons, some files live in gi/
 libgjs_la_SOURCES += \
-	gi/arg.h	\
-	gi/boxed.h	\
-	gi/closure.h	\
-	gi/enumeration.h	\
-	gi/function.h	\
-	gi/keep-alive.h	\
 	gi/gjs_gi_trace.h \
-	gi/ns.h	        \
-	gi/object.h	\
-	gi/foreign.h	\
-	gi/param.h	\
-	gi/repo.h	\
-	gi/union.h	\
-	gi/value.h	\
 	gi/arg.c	\
 	gi/boxed.c	\
 	gi/closure.c	\
--
cgit v0.9
